src/backend/distributed/commands/type.c

```diff
@@ -258,7 +258,7 @@ PreprocessCreateEnumStmt(Node *node, const char *queryString)
 	QualifyTreeNode(node);
 
 	/* reconstruct creation statement in a portable fashion */
-	const char *createEnumStmtSql = DeparseCreateEnumStmt(node);
+	const char *createEnumStmtSql = DeparseTreeNode((Node *) node);
```

Comment:
unclear why this change is made here.

That said, why are we casting to a `Node *` if it is already that type?